src/unix/lwt_engine.mli

```diff
@@ -140,6 +140,16 @@ class libev : ?backend:Ev_backend.t -> unit -> object
     (** Returns [loop]. *)
 end
 
+class libuv : unit -> object
+  inherit t
+
+  val loop : Luv.Loop.t
+    (** The libuv loop used for this engine. *)
+
+  method loop : Luv.Loop.t
```

Comment:
These don't need to be exposed for now, however. I assume this is the result of basing the code on `libev`.